doc/concepts/configuration/configuration_memtx.rst

```diff
@@ -0,0 +1,233 @@
+..  _configuration_memtx:
+
+In-memory storage
+=================
+
+**Example on GitHub**: `memtx <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/config/instances.enabled/memtx>`_
+
+In Tarantool, all data is stored in random-access memory (RAM) by default.
+For this purpose, the :ref:`memtx <engines-memtx>` storage engine is used.
+
+This topic describes how to define settings related to in-memory storage in the
+:ref:`memtx <configuration_reference_memtx>` section of a :ref:`YAML configuration <configuration>`.
```

Comment:
I'd mention a couple basic examples here (for example, memory size and maximum tuple size). In this case, the next note would look better: we tell that you "can configure the memory" and then we tell that "you can estimate the memory size as follows".

Reply:
Updated the description